### ISSUE TITLE
list of package versions.

### ISSUE THREAD

**Issue by agabeg63** [facebookresearch/InterWild#36]
Hello.

I was trying to run the demo.py, but I keep getting the version-related errors. Can you write the list of package versions (conda list, for example)?

Thank you

**Comment by mks0601** [comment]
Could you give your error message? You can just follow this: https://github.com/facebookresearch/pytorch3d/blob/main/INSTALL.md

**Comment by agabeg63** [comment]
i have the following error:
    from pytorch3d import _C
ImportError: libc10_cuda.so: cannot open shared object file: No such file or directory.

I also used for above link for installation

**Comment by mks0601** [comment]
I think there is something wrong with your CUDA installation.

**Comment by agabeg63** [comment]
No, actually it is a pytorch3d error. they have this issue in their github repo. Still couldn't fix it.

**Comment by mks0601** [comment]
Could you follow the instruction in here by creating a virtual env?
https://github.com/facebookresearch/pytorch3d/blob/main/INSTALL.md

**Comment by agabeg63** [comment]
I solved the problem by downgrading to an old pytorch version. Thank you.

**Comment by SSaltEng** [comment]
@agabeg63 - could you share your conda environment list of requirements that ended up working? @mks0601 - could you as well?

Thank you.
